code/datums/ai/_ai_controller.dm

```diff
@@ -87,17 +103,16 @@ have ways of interacting with a specific atom and control it. They posses a blac
 /datum/ai_controller/proc/able_to_run()
 	return TRUE
 
-/// Generates a plan and see if our existing one is still valid.
+
+///Runs any actions that are currently running
 /datum/ai_controller/process(delta_time)
 	if(!able_to_run())
 		walk(pawn, 0) //stop moving
 		return //this should remove them from processing in the future through event-based stuff.
-	if(!current_behaviors?.len)
-		SelectBehaviors(delta_time)
-		if(!current_behaviors?.len)
-			PerformIdleBehavior(delta_time) //Do some stupid shit while we have nothing to do
-			return
 
+	if(!current_behaviors?.len)
```

Comment:
LAZYLEN

Reply:
that will make people think its a lazylist